src/integrationTest/resources/application-intTest.yaml

```diff
@@ -33,6 +33,9 @@ spring:
     multipart:
       max-file-size: 1KB
       max-request-size: 10KB #this affects addAudio as well, so needs to handle a 350mb mp2 file
+  datasource:
```

Comment:
Is the default pool size of 50 causing the tests problems? Just curious. Maybe worth explaining it in a comment.

Reply:
Add comment:

```yaml
maximum-pool-size: 10 # Overrides the value in `application.yaml` to a smaller value more appropriate for integration tests
```